.travis.yml

```diff
@@ -21,7 +21,8 @@ before_script:
 
 script:
 - |
-    travis-cargo build && travis-cargo test
+    travis-cargo build && travis-cargo test &&
+    travis-cargo build -- --features tls && travis-cargo test -- --features tls
 
 after_success:
 - travis-cargo coveralls --no-sudo
```

Comment:
Ideally yes... However, coveralls is broken already in `travis-cargo`. I think there are a few PRs to fix it in their repo. Maybe we should have a separate PR just to remove `travis-cargo`. It may be easier than using that tool.